js/NavMainComponent.js

```diff
@@ -2,21 +2,33 @@
 
 var $ = require('jquery');
 
-function NavMainComponent(html) {
+function NavMainComponent(html, window) {
     this.$html = html;
+    this.$window = $(window);
 };
 
 NavMainComponent.prototype.init = function() {
 
```

Comment:
Could we have consistent spacing here, my preference is no line breaks in the first and last line of a function so 
```
function () {
    // code here without space above or below
}
```